js/app.js

```diff
@@ -2,6 +2,10 @@
 
 var navView = {};
 
+// CLEARING PAGE OF CONTENT EXCEPT SKILLS (EQUATE TO HOME)
+$('.category').not('#skills').hide();
+
+// SINGLE-PAGE NAVIGATION FUNCTION USING JQUERY
```

Comment:
I can't comment on it directly, but the code from lines 9 through 33 is really easy to follow and read. I can immediately understand what each function is doing after being clicked where.